### PR TITLE
Created base RLlib documentation

### PR DESCRIPTION
**Pull request by JenishPatel99**
- Basic documentation for RLlib integration into ULTRA

### CODE REVIEW

File: ultra/docs/getting_started.md [hunk 1]
```diff
@@ -95,7 +97,7 @@ After training your agent, your models should be saved under `logs/<timestamped_
 
 To view the Tensorboard results of this experiment, run the command below:
 ```sh
-$ tensorboard --logdir <abolute_path_to_SMARTS>/logs/<timestamped_experiment_name>
+$ tensorboard --logdir <abolute_path_to_ULTRA>/logs/<timestamped_experiment_name>
```
Comment (AlexLewandowski) [inline ultra/docs/getting_started.md:100]: absolute*

File: ultra/docs/rllib.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# RLlib support in ULTRA
+
+## Summary
+
+We have integrated Ray's Reinforcement Learning framework inside ULTRA to cater users whom are familiar with RLlib. We have blended ULTRA's environment and agent specifications
```
Comment (AlexLewandowski) [inline ultra/docs/rllib.md:5]:
```suggestion
We have integrated Ray's Reinforcement Learning framework inside ULTRA to cater to users who are familiar with RLlib. We have blended ULTRA's environment and agent specifications 
```

File: ultra/docs/rllib.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# RLlib support in ULTRA
+
+## Summary
+
+We have integrated Ray's Reinforcement Learning framework inside ULTRA to cater users whom are familiar with RLlib. We have blended ULTRA's environment and agent specifications 
+inside RLlib's configuration to allow their policies to interact in our environment. We currently support RLlib's PPO policy and the plan is to add provision for SAC, DDPG and DQN.
```
Comment (AlexLewandowski) [inline ultra/docs/rllib.md:6]:
```suggestion
inside RLlib's configuration to allow their policies to interact with the ULTRA environment. We currently support RLlib's PPO policy, but plan to support SAC, DDPG and DQN.
```

File: ultra/docs/rllib.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Setup
+
+To use the RLlib functionality inside ULTRA, you must have the ultra-rl>=0.2. If you have not followed the setup guide for ULTRA, here is a [link](docs/setup.md) to it
```
Comment (AlexLewandowski) [inline ultra/docs/rllib.md:10]:
```suggestion
To use RLlib with ULTRA, you must have the ultra-rl>=0.2. If you have not followed the setup guide for ULTRA, here is a [link](docs/setup.md) to it 
```

File: ultra/docs/rllib.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  - `--level`: The level of the task (default is easy).
+  - `--episodes`: The number of training episodes to run (default is 100).
+  - `--timestep`: The environment timestep in seconds (default is 0.1).
+  - `--headless`: Whether to run training without Envision (default is False).
+  - `--eval-episodes`: The number of evaluation episodes (default is 200).
+  - `--eval-rate`: The rate at which evaluation occurs based on the number of episodes (default is 10000).
+  - `--seed`: The environment seed (default is 2).
+  - `--policy`: The policy (agent) to train (default is ppo). Only PPO is supported for now.
+  - `--log-dir`: The directory to put models, tensorboard data, and training results (default is logs/).
+  - `--max-samples` : The number of samples per iteration (default is 4000)
```
Comment (AlexLewandowski) [inline ultra/docs/rllib.md:36]: max-samples is unclear to me, looking at kimia's PR it looks like its the training batch size? We should probably change the arg name or doc string, but I suppose that comment should be  on the other PR.

**Review by christianjans — COMMENTED** [review]
Looks great! Just some pedantic suggestions on my end.

File: ultra/docs/rllib.md (hunk 5 above)
Comment (JenishPatel99) [reply ultra/docs/rllib.md:36]: Okay, it seems that I had "max-samples" argument when I was doing my tests, yes you are correct, I have changed it to "--training-batch-samples", let me know if you are okay with this :)

**Review by christianjans — APPROVED** [review]

**Review by AlexLewandowski — APPROVED** [review]

**Comment by JenishPatel99** [comment]
Documentation merged into ultra-rllib

**Pull request merged**